src/tests/components/ForecastDetails.test.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastDetails from "../../components/ForecastDetails";
+
+describe("Forecast Details", () => {
+  const validProps = {
+    forecast: {
+      date: 1111111,
+      temperature: {
+        min: 12,
+        max: 22,
+      },
+      wind: {
+        speed: 13,
+        direction: "s",
+      },
+      humidity: 30,
+    },
+  };
+
+  it("renders correctly", () => {
+    const { asFragment } = render(
+      <ForecastDetails
+        forecast={validProps.forecast}
+        date={validProps.forecast.date}
+        humidity={validProps.forecast.humidity}
+        temperature={validProps.forecast.temperature}
+        wind={validProps.forecast.wind}
```

Comment:
If you look at what `ForecastDetails` expects to receive in props, its just the `forecast` object as you have on line 24 here. These other props can be deleted